pages/index.tsx

```diff
@@ -0,0 +1,294 @@
+  color: theme.palette.text.secondary,
+}));
+
+const PlusIcon = createSvgIcon(
+  // credit: plus icon from https://heroicons.com/
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    fill="none"
+    viewBox="0 0 24 24"
+    strokeWidth={1.5}
+    stroke="currentColor"
+  >
+    <path
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      d="M12 4.5v15m7.5-7.5h-15"
+    />
+  </svg>,
+  "Plus"
+);
+interface State extends SnackbarOrigin {
+  open: boolean;
+}
+
+
+//home
+
+const Home: NextPage = () => {
+  const todosCollection = collection(firestore, "todos");
+  const [todos, setTodos] = useState<QueryDocumentSnapshot<DocumentData>[]>([]);
+  
+  const [loading, setLoading] = useState<boolean>(true);
+ 
+  const router = useRouter();
+
+  //mui buttons
+  
+
+ 
+  useEffect(() => {
+    // get the todos
+
+    getTodos();
+    
+
+    
+    // reset loading
+    setTimeout(() => {
+      setLoading(false);
+    }, 2000);
+    
+  }, []);
+
+
+
+
+  const getTodos = async () => {
+    // construct a query to get up to 10 undone todos
+    
+    const todosQuery = query(
+      todosCollection,
+      where("done", "==", false),
+      limit(10)
+    );
+    // get the todos
+    const querySnapshot = await getDocs(todosQuery);
+
+    // map through todos adding them to an array
+    const result: QueryDocumentSnapshot<DocumentData>[] = [];
+    querySnapshot.forEach((snapshot) => {
+      result.push(snapshot);
+    });
+    // set it to state
+    setTodos(result);
+  
+    
+  };
+
+  const updateTodo = async (documentId: string) => {
+    
+    // create a pointer to the Document id
+    const _todo = doc(firestore, `todos/${documentId}`);
+    // update the doc by setting done to true
+   await updateDoc(_todo, {
+      done:true,
+    });
+
+
+    getTodos();
+   
+    // retrieve todos
+   
+   
+   //update alert
+   
+  alert("Successfully Updated")  ; 
+
+  router.reload();
+  }
+  ;
+
+  const deleteTodo = async (documentId: string) => {
+    // create a pointer to the Document id
+    const _todo = doc(firestore, `todos/${documentId}`);
+    // delete the doc
+    await deleteDoc(_todo);
+
+    // retrieve todos
+    getTodos();
+    //delete alert
+   alert("Successfully Deleted");
+   
+  };
+
+  return (
+    
+    <Container sx={{  minHeight:"100vh",background: "" }}>
+      <title>Todos app</title>
+
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          justifyContent: "center",
+
+          bgcolor: "background.paper",
+          boxShadow: 1,
+          borderRadius: 2,
+          p: 2,
+          maxWidth: "30%",
+          margin: "auto", // Center the container horizontally
+          marginTop: "5%", // Add margin top for the vertical spacing
+        }}
+      >
+        <Stack marginTop="2" direction="column" spacing={1}>
+          <h2 id="add-todo">Add your Todos</h2>{" "}
+          
+          <a href="/add-todo">
```

Comment:
Use the router from next navigation here. Also, do not wrap the button with an anchor tag like this. This is bad practice. Instead, you can use `link` from MUI: https://mui.com/material-ui/api/link/ and have the variant as `button`.

But for this, use the router from Next navigation.